### DISCUSSION TITLE
Mastodon support

### DISCUSSION THREAD

**Discussion by robsonsobral** [robertodoering/harpy#664] [Ideas]
It would be great if we could have a combined mastodon + twitter timeline. 

(Probably, you'll say "no", but... to ask doesn't hurt)

Thank you for your attention.

**Comment by dontobi** [comment]
Mastodon integration (in any form) would be nice, but probably difficult to implement.

**Comment by robsonsobral** [comment]
Elon Musk buying made this an auspicious moment for a Mastodon+Twitter integration.

**Comment by scottiefalkon** [comment]
A separate Mastodon client based on Harpy would be ideal. I would eagerly buy that app.

**Reply by robertodoering** [reply]
I might if Twitter or the API ends up dying

**Comment by dnestico** [comment]
This would probably be the only thing that makes me sign up to mastodon tbh, rn it ain't the best UX...

**Comment by kochia4** [comment]
Mastodon is slow, unable to scale, limited, buggy, etc.
I tried it several years ago, it's not worth.
Twitter getting better.

@robertodoering did you try the Mastodon, it's a toxic, unhealthy place, almost all instance are BS managed by arrogant AHs.

Keep the Harpy as an amazing Twitter client.

**Reply by nyabinary** [reply]
"Twitter is getting better" aged like milk.

**Comment by nyabinary** [comment]
@robertodoering You should do this. It would be cool, plus an open-source app supporting an open-source media platform is quite a perfect matchup IMO as well. With Elon Musk's recent decisions, it's in question if the API or even Twitter as a whole survives.
And I say this as a person who bought Harpy Pro.

**Comment by nyabinary** [comment]
Also want to note that an alternative Twitter client (like Harpy) also implemented Mastodon support. https://play.google.com/store/apps/details?id=org.mariotaku.twidere
With Elon Musk banning journalist I'm jumping ship to Mastodon myself honestly.
